### PR TITLE
Add SQL server devops audit settings resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

**Comment by irmatov** [comment]
Work in progress/ to be reviewed/ amended after previous PRs.

**Review by fdistorted — APPROVED** [review]
Looks good

File: resources/sql_servers.go [hunk 1]
```diff
@@ -364,6 +364,102 @@ func SQLServers() *schema.Table {
...
+					},
+					{
+						Name:        "last_modified_at_time",
+						Description: "The timestamp of last modification (UTC).",
+						Type:        schema.TypeTimestamp,
+						Resolver:    schema.PathResolver("SystemData.LastModifiedAt.Time"),
+					},
+					{
+						Name:        "is_azure_monitor_target_enabled",
+						Description: "Specifies whether DevOps audit events are sent to Azure Monitor In order to send the events to Azure Monitor, specify 'State' as 'Enabled' and 'IsAzureMonitorTargetEnabled' as true  When using REST API to configure DevOps audit, Diagnostic Settings with 'DevOpsOperationsAudit' diagnostic logs category on the master database should be also created  Diagnostic Settings URI format: PUT https://managementazurecom/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/MicrosoftSql/servers/{serverName}/databases/master/providers/microsoftinsights/diagnosticSettings/{settingsName}?api-version=2017-05-01-preview  For more information, see [Diagnostic Settings REST API](https://gomicrosoftcom/fwlink/?linkid=2033207) or [Diagnostic Settings PowerShell](https://gomicrosoftcom/fwlink/?linkid=2033043)",
```
Comment (roneli) [inline resources/sql_servers.go:417]: shorten description

**Review by roneli — APPROVED** [review]
LGTM, minor comment and merge

File: resources/sql_servers.go (hunk 1 above)
Comment (irmatov) [reply resources/sql_servers.go]: Done

**Pull request merged**